domains/razin.json

```diff
@@ -6,5 +6,7 @@
 
   "record": {
     "CNAME": "razin.pages.dev"
+    "MX": ["mx1.forwardemail.net", "mx2.forwardemail.net"],
+    "TXT": "forward-email=razin.rayeesk@gmail.com"
```

Comment:
You can't have any other records with CNAME, you can use A records instead of CNAME if you want to use other records.